model_compression_toolkit/exporter/model_wrapper/__init__.py

```diff
@@ -13,8 +13,12 @@
 # limitations under the License.
 # ==============================================================================
 
-from model_compression_toolkit.exporter.model_wrapper.keras.validate_layer import is_keras_layer_exportable
-from model_compression_toolkit.exporter.model_wrapper.keras.builder.fully_quantized_model_builder import get_exportable_keras_model
+from mct_quantizers.common import constants
```

Comment:
These changes are not relevant to this PR.
Please remove the changes of this file from this changeset.

Reply:
This phase is extremely slow and can easily take more than a coffee break.
I think it's critical to show users something is still going on in such cases.

Reply:
I think your comment here is meant for somewhere else? (the adding of tqdm?)
If so, I with that, but the changes in this file are not related to this PR.